package.json

```diff
@@ -11,30 +11,32 @@
     "react-component"
   ],
   "dependencies": {
-    "dehumanize-date": "^2.0.1",
-    "occasion": "^1.0.0",
-    "prop-types": "^15.5.8"
+    "dehumanize-date": "2.4.0",
+    "occasion": "2.0.0",
+    "prop-types": "^15.5.10",
+    "react-dom": "^15.6.1",
+    "react-test-renderer": "^15.6.1"
```

Comment:
Thanks. With the removal of `package.json` I made sure all packages have `^` in front of them again for semver instead of lockdown.